omc3/optics_measurements/rdt.py

```diff
@@ -209,7 +249,11 @@ def get_line_sign_and_suffix(line, input_files, plane):
         return 1, suffix
     if f"AMP{conj_suffix}" in input_files[plane][0].columns:
         return -1, conj_suffix
-    raise ValueError(f"No data for line {line} in plane {plane}")
+
+    # The specified AMP column hasn't been found in the lin file
+    msg = (f"The column AMP{suffix} has not been found in the lin{plane.lower()} file. "
+            "Consider re-running the frequency analysis with a higher order resonance term")
```

Comment:
What flag for `harpy` would that correspond to?

Reply:
There are no flags, it is referencing this line `RESONANCES = get_resonance_lines(5)`
I'm not sure how to implement that with a flag at the moment

Reply:
Yes. It's a bit more involved as it should be a common parameter in both harpy and measure optics. And I think if not given it should default to something lower though. 
As mael was quite busy we said we can introduce this in two steps (i.e. two PRs) but as we want. Important here would be to make sure the principle is correct.

Reply:
(Also, the resonances dict could then be just a function instead of a global parameter)

Reply:
I agree this can ship as is for now. I don't think it matters too much to make it a lower value by default as it only kicks in if `--rdt` is provided to `measure_optics`.

Reply:
that is the problem: it also kicks in in harpy. So the lin-files get bigger. That's also why you need this flag in both

Reply:
I just added arguments for both harpy and the optics:
* harpy: --resonances
* optics: --rdt_magnet_order, to avoid confusing it with the order of the RDT